eoscostarica/eosio-dashboard
# 862 improve position of recaptcha

### Fix Captcha Position

### What does this PR do?

- Resolve #862 

### Steps to test

1. Navigate to localhost and see footer & see captcha on mainnet

Keep in mind its the old footer
Collapsed:
![image](https://user-images.githubusercontent.com/55892352/190013293-319301a5-dbdd-4375-9c47-bb1fe0b7a342.png)

Extended:
![image](https://user-images.githubusercontent.com/55892352/190013390-1080b884-4187-41be-ac36-e799d028bb79.png)


#### CheckList
